docs/features/aliases.md

```diff
@@ -1,5 +1,6 @@
 # Type Aliases
 
+Type Aliases are great ways of compessing types into a more friendly form for usage in functions and other type declarations. It can be used as so:
```

Comment:
```suggestion
Type aliases are great ways of compressing types into a more friendly form for usage in functions and other type declarations. It can be used as so:
```